ipaserver/install/kra.py

```diff
@@ -69,9 +68,16 @@ def install_check(api, replica_config, options):
 
 
 def install(api, replica_config, options):
-    if replica_config is None:
-        if not options.setup_kra:
-            return
+
+    if replica_config is not None and not replica_config.setup_kra:
+        return
+
```

Comment:
About the above, although in the server install code we calll kra.install only if options.setup_kra is true, it feels like we may just want to check:
if not options.setup_kra:
    return

This will catch either replica or server install cases should we change how installs are called in future.
In fact we might even go to the server install code and skip the check there and let the install return.

Note that to do this we'll need to change replicainstall.py to set:
installer.setup_kra = options.setup_kra
instead of:
installer._kra_enabled = kra_enabled